runtime/nls/cfre/cfrerr.nls

```diff
@@ -1750,3 +1750,11 @@ J9NLS_CFR_ERR_NULLRESTRICTED_ATTRIBUTES_LENGTH_IS_ZERO.explanation=Please consul
 J9NLS_CFR_ERR_NULLRESTRICTED_ATTRIBUTES_LENGTH_IS_ZERO.system_action=The JVM will throw a verification or classloading related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_NULLRESTRICTED_ATTRIBUTES_LENGTH_IS_ZERO.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
+
+# ImplicitCreation is not translatable
+J9NLS_CFR_ERR_IMPLICITCREATION_ILLEGAL_CLASS_MODIFIERS=The ImplicitCreation attribute is only allowed in a non-abstract value class
```

Comment:
I see the message name is still `_ILLEGAL_CLASS_MODIFIERS`. As the message changed, I believe you need a new name for the message.

Reply:
To me it describes the gist of the issue, what would make it more clear?

Reply:
OK, I am good with it.